### DISCUSSION TITLE
How to properly use of step in lr_config?

### DISCUSSION THREAD

**Discussion by sarmientoj24** [open-mmlab/mmdetection#9163] [Q&A] [answered]
Been seeing `step` in `lr_config` where the step, for example, is `[8, 11]` and the `epochs` are `12`.

How do you properly configure that step? Say, the epoch I put is 75. What should be the step?

**Comment by jbwang1997** [comment]
It doesn't have a rule to set the step. The step used in mmdetection is as same as it in detectron2. 

In your situation, you can simply linearly expand the [8, 11, 12] step schedule to totally 75 epochs.

**Comment by sarmientoj24** [comment]
So the `step_config` for 75 epochs is still `[8, 11]`? I thought the epoch has influence on the step?

**Comment by jbwang1997** [comment]
You can use [50, 69] which is equal to [8, 11] x (75/12).

**Comment by ZwwWayne** [comment, marked as answer]
The steps are usually empirically tuned for different epochs to reduce the overfitting and achieve better performance. We suggest you follow the existing configs of supported datasets and models in MMDetection.
